go/consensus/tendermint/api/context.go

```diff
@@ -103,6 +105,7 @@ func NewContext(
 	blockHeight int64,
 	blockCtx *BlockContext,
 	initialHeight int64,
+	priority int64,
```

Comment:
We could remove the constructor argument and assume if anyone wants to set priority then `SetPriority` would be called after construction.